### DISCUSSION TITLE
Delete Order

### DISCUSSION THREAD

**Discussion by JoeAucoin** [openstore-ecommerce/OpenStore#271] [Q&A]
Am I missing something? Is it possible to delete an order?

**Comment by DNNMonster** [comment]
The OS admin can delete all the orders to reset a system from the Back Office...  In general it's not acceptable to allow a single order to be deleted as it would also delete the audit log for the transaction which is bad.

Once a system is up an running the UI will only let you set an order status to Cancelled if needed.  No deleting.

If you're in a dev or test environment then there are multiple ways to run SQL against the db if you have a need to remove specific transactions.   You can create a custom action plugin, use the DNN SQL tool, or connect directly to the db with SSMS.  

The same approaches can be applied to production but it's not advisable for previously mentioned reasons.

**Reply by leedavi** [reply]
The reason the delete button was removed was that one client was claiming orders were disappearing, we had no proof it was them deleting the orders.  I spent a lot of time putting debug statements in to find out it was one of them deleting the orders.
